tests/lib/config.js

```diff
@@ -776,7 +776,7 @@ describe("Config", () => {
         });
 
         describe("with env in a child configuration file", () => {
-            it("should overwrite parserOptions of the parent with env of the child", () => {
+            xit("should overwrite parserOptions of the parent with env of the child", () => {
```

Comment:
Skipping this is fine for now, but at some point we should make a decision about the intended behavior and write tests to make sure it doesn't change.